contrib/platform/test/com/sun/jna/platform/win32/COM/ShellApplicationWindowsTest.java

```diff
@@ -0,0 +1,232 @@
+package com.sun.jna.platform.win32.COM;
+
+import java.util.Iterator;
+import java.util.NoSuchElementException;
+
+import com.sun.jna.Native;
+import com.sun.jna.NativeLibrary;
+import com.sun.jna.platform.win32.OleAuto;
+import com.sun.jna.platform.win32.Variant;
+import com.sun.jna.platform.win32.Variant.VARIANT;
+import com.sun.jna.platform.win32.WinDef.LONG;
+import com.sun.jna.platform.win32.WinDef.LONGLONG;
+
+import junit.framework.TestCase;
+
+public class ShellApplicationWindowsTest extends TestCase {
+
+	@Override
+	protected void setUp() throws Exception {
+		String[] commands = {"cmd", "/c", "start", "iexplore.exe","-nohome", "\"http://www.srh.noaa.gov/jax\""};
+		Runtime.getRuntime().exec(commands);
+		Thread.sleep(30000);
+	}
+
+	public void testWindowsCount() {
+        System.out.println("java.vendor: " + System.getProperty("java.vendor"));
+        System.out.println("java.version: " + System.getProperty("java.version"));
+        System.out.println("os.version: " + System.getProperty("os.version"));
+        System.out.println("os.arch: " + System.getProperty("os.arch"));
+        System.out.println("os.name: " + System.getProperty("os.name"));
+        
+        ShellApplication sa = new ShellApplication();
+        System.out.println("Shell.Application.Windows found: " + sa.Windows().Count());
+        assertTrue(sa.Windows().Count() > 0);
+        for (InternetExplorer ie : sa.Windows())
+        {            
+            System.out.println();
+            System.out.println("============================================");
+            if (ie != null)
+            {
+                System.out.println("Title: " + ie.getTitle());
+                System.out.println("URL: " + ie.getURL());
+                System.out.println("App Name: " + ie.getName());
+                System.out.println("Window Handle: " + ie.getWindowHandle());
+            }
+            else
+            {
+                System.out.println("Null encountered.");
+            }
+            System.out.println("============================================");
+            System.out.println();
+        }
```

Comment:
On all my test boxes I get a bunch of nulls in addition to the IE window I want. 
I'm not sure why -- MSDN says nothing about Shell.Application.Windows.Item[x] returning null. 

As long as I saw the window I wanted get printed out, I was happy - so I made the initial program ignore null entries.